src/components/project/form-row.vue

```diff
@@ -108,6 +111,10 @@ export default {
     project: {
       type: Object,
       required: true
+    },
+    showIcon: {
```

Comment:
I was confused about this at first - could you add a comment describing what this is for, how it's to place the icon in the form/dataset sidebar thing by having it be in the first row?